servers/src/test/java/tachyon/master/file/meta/InodeDirectoryTest.java

```diff
@@ -183,4 +184,25 @@ public void getChildTest() {
         System.currentTimeMillis() - start));
   }
 
+  @Test
+  public void generateClientFileInfoTest() {
+	  InodeDirectory inodeDirectory = createInodeDirectory();
```

Comment:
In tachyon codebase, we use spaces (not tabs) and it should be indented 2 spaces. Thanks!